### PR TITLE
Remove `:change_of_circumstances` feature flag

### PR DESCRIPTION
**Pull request by peteryates**
### Context

This is a WIP. The flag hasn't been toggled for a _long_ time and some of our tests assume it's enabled and other bits assume it isn't.

### Changes proposed in this pull request

* remove all references to the `:change_of_circumstances` feature flag from the app
* update some specs that assume the flag was disabled despite it being enabled in production

### Suggested code review apporach

I'd go through commit by commit, [the final one](https://github.com/DFE-Digital/early-careers-framework/pull/3017/commits/aa6b68239f21264fbdd018c0b80144a0f5106958) is a big and messy mass removal and will confuse any reviewing.

### CODE REVIEW

File: spec/requests/admin/participants_spec.rb [hunk 1]
```diff
@@ -23,18 +23,22 @@
 
     it "only includes active participants" do
```
Comment (ltello) [inline spec/requests/admin/participants_spec.rb:24]: So this is not true anymore no?

Comment (peteryates) [reply spec/requests/admin/participants_spec.rb:24]: It is not 😩

Comment (peteryates) [reply spec/requests/admin/participants_spec.rb]: I reworded the descriptions and re-pushed. Thanks @ltello

**Comment by ltello** [comment]
Wonderful. I love when old code goes away!

**Comment by ltello** [comment]
...cleaning
excellent!

**Review by ltello — APPROVED** [review]

**Pull request merged**